b2b-sample/guardio-insurance-sample-application/pages/api/settings/role/createRole.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { getSentDataRequestOptions } from "../../../../util/util/apiUtil/getSentDataRequestOptions";
+import { dataNotRecievedError, notPostError } from "../../../../util/util/apiUtil/localResErrors";
+import { RequestMethod } from "../../../../util/util/apiUtil/requestMethod";
+import { getOrgUrl } from "../../../../util/util/orgUtil/orgUtil";
+
+/**
+ * backend API call to list all roles.
```

Comment:
Let's start the comments with upper case letter and ends with a full-stop. Check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b